src/hosted-buttons/index.js

```diff
@@ -19,12 +25,14 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
   function HostedButtons({
     enableDPoP = false,
     hostedButtonId,
+    fundingSources = [],
   }: HostedButtonsComponentProps): HostedButtonsInstance {
     const Buttons = getButtonsComponent();
     const render = async (selector) => {
       const merchantId = getMerchantID();
       const { html, htmlScript, style } = await getHostedButtonDetails({
         hostedButtonId,
+        fundingSources,
```

Comment:
I see you're passing `fundingSources` to `getHostedButtonDetails()`, but `getHostedButtonDetails()` doesn't seem to be doing anything with it. 🤔  https://github.com/paypal/paypal-checkout-components/pull/2375/files#diff-dc0ab5efddb0f4c5dc8c699ccafce1daa87906e1b839480c5be57048cee5482bR82-R84

Reply:
Flow was complaining about it being missing when I removed it as an option prop from `HostedButtonsComponentProps`.